se_feedback/sprint_6/feedback_sprint_6_all.md

```diff
@@ -61,3 +61,23 @@ Total: ****
 | Code Review            |     |
 
 ##Notes
+* ADD
+	* The diagrams makes your explanation a lot clearer.
+	* You should explain how you run the tools in the Concurrency section 
+* Good job on the test coverage document!
+* Java
+	* `JavaController.setASATOutput`
+		* Consider using a properties file (or JSON) to store ASAT Output locations
```

Comment:
@ClintonCao It seems that I've misunderstood what this code does. ASAT Output locations refers to the locations of the output files generated by the ASATs (the xml files to be parsed)?

If so, then this comment isn't relevant.